since this will provide the same `.so` as real harfbuzz, make apk strongly
disprefer this one by lowering the minor version in the .so.
this is only used for freetype
diff --git a/meson.build b/meson.build
index 8d1c732..52fcdfd 100644
--- a/meson.build
+++ b/meson.build
@@ -15,7 +15,7 @@ hb_version_minor = hb_version_arr[1].to_int()
 hb_version_micro = hb_version_arr[2].to_int()
 
 # libtool versioning
-hb_version_int = 60000 + hb_version_major*100 + hb_version_minor*10 + hb_version_micro
+hb_version_int = 1
 hb_libtool_version_info = '@0@:0:@0@'.format(hb_version_int)
 
 pkgmod = import('pkgconfig')
